components/brave_wallet/renderer/js_ethereum_provider.cc

```diff
@@ -107,46 +98,46 @@ void BraveWalletJSHandler::SendResponse(
   }
 }
 
-BraveWalletJSHandler::BraveWalletJSHandler(content::RenderFrame* render_frame,
-                                           bool brave_use_native_wallet,
-                                           bool allow_overwrite_window_ethereum)
+JSEthereumProvider::JSEthereumProvider(content::RenderFrame* render_frame,
+                                       bool brave_use_native_wallet,
+                                       bool allow_overwrite_window_ethereum)
     : render_frame_(render_frame),
       brave_use_native_wallet_(brave_use_native_wallet),
       allow_overwrite_window_ethereum_(allow_overwrite_window_ethereum),
       is_connected_(false) {
   if (g_provider_script->empty()) {
-    *g_provider_script =
-        LoadDataResource(IDR_BRAVE_WALLET_SCRIPT_BRAVE_WALLET_SCRIPT_BUNDLE_JS);
+    *g_provider_script = LoadDataResource(
+        IDR_BRAVE_WALLET_SCRIPT_ETHEREUM_PROVIDER_SCRIPT_BUNDLE_JS);
```

Comment:
This change will have to be applied to iOS as well:

https://github.com/brave/brave-core/blob/ef9cae844e7cedc7a6f19f1d4f5521268cf18475/ios/app/brave_core_main.mm#L340

Reply:
yep, just saw CI build failure on iOS

Reply:
fixed